scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobEmailNotification.java

```diff
@@ -262,9 +282,11 @@ private static String getFrom() throws JobEmailNotificationException {
         return Arrays.asList(toList);
     }
 
-    private String getSubject(boolean withErrors) {
+    private String getSubject(boolean withErrors, boolean aborted) {
         String event = withErrors ? SchedulerEvent.JOB_RUNNING_TO_FINISHED_WITH_ERRORS.toString()
                                   : eventType.toString();
+        if (aborted)
```

Comment:
To improve code clarity, consider refactoring the ternary operator above.

Reply:
is this more clear? 
```
        String event = eventType.toString();
        if (withErrors)
            event= SchedulerEvent.JOB_RUNNING_TO_FINISHED_WITH_ERRORS.toString();
        if (aborted)
            event = SchedulerEvent.JOB_ABORTED.toString();
```

Reply:
Yes